### ISSUE TITLE
RAPT running failure

### ISSUE THREAD

**Issue by mglgc** [ncbi/rapt#5]
The ./run_rapt.py --test command outputs the below error message contained in the verbose.log file.
Please, could the developers contribute with any hints to fix it? Thank you in advance.

[2020-11-21 00:34:08] Sending PINGER url https://www.ncbi.nlm.nih.gov/stat?ncbi_app=raptdocker&version=2020-09-24.build4894&uuid=43cbaa1a-2b91-11eb-ada1-901b0e950242&evt=rapt_start&
[2020-11-21 00:34:48] rapt-29571188
-------------------------------------------------

[2020-11-21 00:34:48] Sending PINGER url https://www.ncbi.nlm.nih.gov/stat?ncbi_app=raptdocker&version=2020-09-24.build4894&uuid=43cbaa1a-2b91-11eb-ada1-901b0e950242&evt=test_start
[2020-11-21 00:35:29] Running functional tests...
[2020-11-21 00:35:30] problem with tcp access to ncbi.gov sdl host

[2020-11-21 00:35:30] port closed

[2020-11-21 00:35:30] SRA connection check failed with code 1, abort..
[2020-11-21 00:35:30] Sending PINGER url https://www.ncbi.nlm.nih.gov/stat?ncbi_app=raptdocker&version=2020-09-24.build4894&uuid=43cbaa1a-2b91-11eb-ada1-901b0e950242&evt=test_end
[2020-11-21 00:36:10] status=1
[2020-11-21 00:36:10] Sending PINGER url https://www.ncbi.nlm.nih.gov/stat?ncbi_app=raptdocker&version=2020-09-24.build4894&uuid=43cbaa1a-2b91-11eb-ada1-901b0e950242&evt=rapt_exit

**Comment by thibaudnis** [comment]
Oops, looks like this issue went unnoticed. Thank you for the report. We will get back to you in the next few days.

**Comment by lwagnerdc** [comment]
Thanks for writing with a clear description of your problem.  

Does the computer where you ran this have a firewall in place?  If yes, it will be necessary to allow egress to the NCBI network services used by rapt for SRA access.   
In particular, from the error message, port 443 of 130.14.29.110  looks to have been unavailable.  

A complete list of SRA network resources is available here: 
https://github.com/ncbi/sra-tools/wiki/Firewall-and-Routing-Information
RAPT may use services 6-8 on the list of SRA resources; under ordinary circumstances, only items 6 and 7 will see traffic.   If changing your firewall is non-trivial (for example, if you need to communicate with a systems group who may not respond immediately), it's safest to request access to the services listed as items 6,7,8 there.  

If this does not help, please let us know, be assured that the RAPT team is paying attention to error reports here now.

**Comment by mglgc** [comment]
> Thanks for writing with a clear description of your problem.
> 
> Does the computer where you ran this have a firewall in place? If yes, it will be necessary to allow egress to the NCBI network services used by rapt for SRA access.
> In particular, from the error message, port 443 of 130.14.29.110 looks to have been unavailable.
> 
> A complete list of SRA network resources is available here:
> https://github.com/ncbi/sra-tools/wiki/Firewall-and-Routing-Information
> RAPT may use services 6-8 on the list of SRA resources; under ordinary circumstances, only items 6 and 7 will see traffic. If changing your firewall is non-trivial (for example, if you need to communicate with a systems group who may not respond immediately), it's safest to request access to the services listed as items 6,7,8 there.
> 
> If this does not help, please let us know, be assured that the RAPT team is paying attention to error reports here now.

**Comment by mglgc** [comment]
> Thanks for writing with a clear description of your problem.
> 
> Does the computer where you ran this have a firewall in place? If yes, it will be necessary to allow egress to the NCBI network services used by rapt for SRA access.
> In particular, from the error message, port 443 of 130.14.29.110 looks to have been unavailable.
> 
> A complete list of SRA network resources is available here:
> https://github.com/ncbi/sra-tools/wiki/Firewall-and-Routing-Information
> RAPT may use services 6-8 on the list of SRA resources; under ordinary circumstances, only items 6 and 7 will see traffic. If changing your firewall is non-trivial (for example, if you need to communicate with a systems group who may not respond immediately), it's safest to request access to the services listed as items 6,7,8 there.
> 
> If this does not help, please let us know, be assured that the RAPT team is paying attention to error reports here now.

**Comment by mglgc** [comment]
Thanks for the reply. A bit late to be useful, but some reply at last.
If a get it properly, have you saying that one undocumented prerequisite for a successful stand-alone RAPT installation is have to open any local network firewall to the services #6-8 on the list of SRA resources? If so, I would suggest that this requirement should be included as key information in the program documentation for a successful RAPT running.
Thank you in advance for your reply.

**Comment by lwagnerdc** [comment]
Yes, access to SRA's network services for data retrieval is a prerequisite for running RAPT.
You raise a good point that this should be written clearly in the documentation.  Thank you for the suggestion, we'll improve it.

**Comment by mglgc** [comment]
We are trying to run the standalone version of RAPT under Ubuntu 16.04 but the next error message is got after
./run_rapt.py --test:
"Cannot find docker compatible program".

Please, could you point us to any fix? Thank you so much in advance.

**Comment by lwagnerdc** [comment]
Hello, is there a docker binary on your system? 

"which docker" will check for the presence of a binary with that name everywhere covered by your environment's $PATH  .  If you're running on a shared environment, your systems group may have prepared instructions for  users interested in docker without enabling it by default, or have a policy in place about using docker.
If you're running on a machine you control, then installing docker will likely be your next step. I can look for guidelines specific to your environment in that case; knowing more about your environment would be helpful if this is relevant.

**Comment by mglgc** [comment]
Thank you for your prompt reply. So, one additional prerequisity is to have a docker installation to run the standalone RAPT version, isn't it?

**Comment by lwagnerdc** [comment]
Yes, a container runner is necessary, with docker preferred. 

I think that this is mentioned in the documentation we have now for standalone RAPT: 
https://github.com/ncbi/rapt/blob/master/Standalone%20RAPT.md

**Comment by Yoandris90** [comment]
Hola al ejecutar el RAPT en mi terminal ubuntu me genera este mensaje: -rw-r--r-- 1 gpipe gpipe 0 Dec 29 14:25 stdcap.bin
